highfidelity/hifi
# Add local-only isVisibleInSecondaryCamera flag to entities

(shouldn't require a protocol bump)

Adds a local flag to all entity types that let's you disable their rendering in the secondary camera view while continuing to render them in the main view.

https://highfidelity.manuscript.com/f/cases/12763/camera-flicker-when-flying-was-Spectator-Cam-issues-from-Phlash

Test plan:
- Stand in front of a mirror.
- Run [this](https://gist.githubusercontent.com/SamGondelman/a44a950be3d513933c0e7ec70e01c556/raw/64774f1ec9c5cf9ad5c93d09ebce6a3beb981f6d/VisibleInSecondaryCameraTest.js).
- You should see a white box, two hats, and a green wireframe box, in both the main view and mirror.
- Press 'u'.  They should all disappear in the main view and the mirror.  Press it again and they should come back.
- Press 'i'.  They should disappear in the mirror, but remain in the main view.  Press it again and they should come back in the mirror.
- In first person, you should be able to see your head in the mirror.  When you walk, your eyes shouldn't appear in your view.  In third person, your head should appear in the main view and the mirror.
